homeassistant/components/sensor/rest.py

```diff
@@ -156,6 +156,9 @@ def update(self):
                         attrs = {k: json_dict[k] for k in self._json_attrs
                                  if k in json_dict}
                         self._attributes = attrs
+                        attrs_str = json.dumps(attrs)
+                        value = (attrs_str[:252] + '...') \
```

Comment:
This makes no sense. We're overriding the value and truncating it. Instead of trying to "fix" the data, we should throw an error and mark entity as unavailable.